pkg/clusteragent/externalmetrics/provider_test.go

```diff
@@ -194,6 +195,31 @@ func TestGetExternalMetrics(t *testing.T) {
 			expectedExternalMetrics: nil,
 			expectedError:           fmt.Errorf("DatadogMetric not found for metric name: nginx.net.request_per_s, datadogmetricid: default/dcaautogen-32402d8dfc05cf540928a606d78ed68c0607f7"),
 		},
+		{
+			desc: "Test CustomExternalMetric - DatadogMetric exists and is valid",
```

Comment:
You may want to change the desc to something like "case sensitive metric name" or something

Reply:
I will do that, thanks for the hint

Reply:
Done